src/components/MainComponent.vue

```diff
@@ -73,6 +73,9 @@ export default defineComponent({
     },
     capitalizedItemName() {
       return this.itemName.charAt(0).toUpperCase() + this.itemName.slice(1);
+    },
+    mockedDataMessage() {
+      return process.env.VUE_APP_MOCKED_DATA === 'true' ? 'Using mock data. Check VUE_APP_MOCKED_DATA variable.' : '';
```

Comment:
```suggestion
      return process.env.VUE_APP_MOCKED_DATA === 'true' ? 'Using mock data - see README if unintended' : '';
```

Reply:
@abirismyname what do you think about the suggestion? Thanks for your contribution!